test/screens/v2/departure_test.exs

```diff
@@ -196,15 +196,15 @@ defmodule Screens.V2.DepartureTest do
 
   describe "route_name/1" do
     test "returns prediction route short_name when present" do
-      prediction = %Prediction{route: %Route{id: "214216", short_name: "214/216"}}
+      prediction = %Prediction{route: %Route{type: :bus, short_name: "214/216"}}
       schedule = %Schedule{route: %Route{id: "1", short_name: "1"}}
       departure = %Departure{prediction: prediction, schedule: schedule}
 
       assert "214/216" == Departure.route_name(departure)
     end
 
     test "returns schedule route short_name when no prediction is present" do
-      schedule = %Schedule{route: %Route{id: "214216", short_name: "214/216"}}
+      schedule = %Schedule{route: %Route{type: :bus, short_name: "214/216"}}
```

Comment:
can you add the breaking case that was causing this bug? 'sl2' or something like that for the id